docs/release-notes/november-2023.md

```diff
@@ -90,25 +91,25 @@ If no issues are detected after a few days, the Semgrep team then promotes the `
 ### Fixed
 
 -  **Slack messages:**
-    - Improved readability of Semgrep Supply Chain messages by adding newlines between sections. <!-- (11396) -->
+    - Improved readability of Semgrep Supply Chain messages by adding new lines between sections. <!-- (11396) -->
     -  Fixed links that were not working. <!-- (11210) -->
-- Fixed out of bounds list access error in `Cargo.lock` parser. <!-- (sc-1072) -->
+- Fixed out-of-bounds list access error in `Cargo.lock` parser. <!-- (sc-1072) -->
 
 ## 🔐 Semgrep Secrets (beta)
 
 ### Added
 
-- Added an optional `--no-secrets-validation` flag to skip secrets validation. To run a secrets scan with no validation, enter `semgrep ci --secrets --no-secrets-validation`.
+- Added an optional `--no-secrets-validation` flag to skip secrets validation. To run a secrets scan without validation, enter `semgrep ci --secrets --no-secrets-validation`.
 - **Secrets and Secrets details page:** Added a **<i class="fa-solid fa-ticket"></i> ticket icon** to quickly inform users if a ticket has been created for the finding.
 
 ### Changed
 
-- Semgrep Secrets now bypasses `.semgrepignore`. This means that files not typically part of a SAST or SCA scan scope, such as configuration files, are now scanned by Semgrep Secrets. By broadening the scope of Semgrep Secrets scans, it is likelier to find leaked secrets.
-    - Previously, Semgrep Secrets excluded targets from `.semgrepignore`. Your findings count may increase with this change.
+- Semgrep Secrets now bypasses targets indicated in `.semgrepignore`. This means that files not typically part of a SAST or SCA scan scope, such as configuration files, are now scanned by Semgrep Secrets. Broadening the scope of Semgrep Secrets scans means it is more likely to find leaked secrets.
+    - Because Semgrep Secrets no longer excludes targets from `.semgrepignore`, your findings count may increase with this change.
```

Comment:
I'm choosing to keep "previously" here as it's a common and familiar convention.